### ISSUE TITLE
Family Addresses Not Importing

### ISSUE THREAD

**Issue by jadtaylor** [NewSpring/Excavator#126]
Everything successfully imported into Rock from our CSV dataset except the family addresses. They did not import at all. I have included a screenshot of the first line of our Family.csv file. I am happy to provide any further information which would be helpful. Thanks in advance!

![family csv first line](https://cloud.githubusercontent.com/assets/6354794/7940131/e6e94cd6-091c-11e5-9335-00be9bc6b6fe.png)

**Comment by dcs619** [comment]
What version of Rock are you running?  There is an address fix in Rock 1.3.4 that is probably related to this issue.

**Comment by jadtaylor** [comment]
We imported the data in February, so we would have been on McKinley 2.0. So, you suggest we should wait for the next release to import again, if I'm reading you right?

**Comment by arranf** [comment]
If you're using the latest beta release 1.3.4 then you'll have to wait for Excavator to get updated. Otherwise Excavator should work.

**Comment by jadtaylor** [comment]
We're running 1.3.3, so I will try another import soon and report back.

**Comment by dcs619** [comment]
If you're running a production environment I would wait for 1.3.4.  The core Rock service is supposed to look up addresses and prevent duplicates but it was crashing in 1.3.3.  Excavator works with 1.3.3, but waits for the Rock lookup to complete (and tries the address again if it crashed).

**TL;DR** 1.3.3 will work but takes longer and has a small chance of duplicate addresses.

**Comment by jadtaylor** [comment]
We just upgraded to 1.3.4. Should we wait for Excavator to be updated before running a test import? I wasn't sure after Arran's comment.

**Comment by dcs619** [comment]
If you're not running in production, you can use the develop branch version of Excavator instead of the master/release branch.  

Each Rock update changes the database significantly (affecting the import), hence Arran's comment.

**Comment by dcs619** [comment]
@jadtaylor Rock 1.3.4 was released over the weekend and Excavator is updated.

**Comment by dcs619** [comment]
Closing as the address issue is fixed in 1.4
